src/PowerTools/DbContextPackage.cs

```diff
@@ -520,7 +520,7 @@ private static void SetDataDirectory(Project project)
                 "DataDirectory",
                 project.IsWebProject()
                     ? Path.Combine(project.GetProjectDir(), "App_Data")
-                    : project.GetTargetDir());
+                    : Path.GetFullPath(project.GetTargetDir()));
```

Comment:
I am not sure it will ever be necessary for WebSites but I can doing over the whole ternary expression. 

I could do it on the extension methods but setting the data directory is the only place where I know path traversals are not supported.